assets/operator/ocp-x86_64/fis/imagestreams/fuse7-karaf-openshift-rhel7.json

```diff
@@ -135,18 +135,18 @@
 				}
 			},
 			{
-				"name": "1.7",
+				"name": "1.6",
```

Comment:
fuse-karaf 1.7 tag is not removed
1.7
  tagged from registry.redhat.io/fuse7/fuse-karaf-openshift:1.7
    prefer registry pullthrough when referencing this tag

  Red Hat Fuse 7.7 Karaf S2I images.
  Tags: builder, jboss-fuse, java, karaf, xpaas, hidden
  Supports: jboss-fuse:7.7.0, java:8, xpaas:1.2

  * registry.redhat.io/fuse7/fuse-karaf-openshift@sha256:4ffd7ccbe8ff0aa2e09e1c8a72410aadc721ed3ed227890f03ce3f8aa2b33700
      33 minutes ago

Reply:
again good catch @xiuwang thanks

list in description updated @rolfedh

Reply:
> again good catch @xiuwang thanks
> 
> list in description updated @rolfedh

ack

Reply:
I've created https://issues.redhat.com/browse/RHDEVDOCS-2931 to track the release notes for this change.